### PR TITLE
feat(locale): Add bio, animal, word, music for zh_CN

### PR DESCRIPTION
**Pull request by CoolPlayLin**
# Works
- Add bio module for zh_CN
- Add animal module for zh_CN
- Add word module for zh_CN
- Add music module for zh_CN

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Comment by CoolPlayLin** [comment]
@ST-DDT It appears the first failure is due to action itself

File: src/locales/zh_CN/person/bio_pattern.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default [
+  '{{person.bio_part}}',
+  '{{person.bio_part}}, {{person.bio_part}}',
+  '{{person.bio_part}}, {{person.bio_part}}, {{person.bio_part}}',
+  '{{person.bio_part}}, {{person.bio_part}}, {{person.bio_part}} {{internet.emoji}}',
+  '{{word.noun}} {{person.bio_supporter}}',
+  '{{word.noun}} {{person.bio_supporter}}  {{internet.emoji}}',
+  '{{word.noun}} {{person.bio_supporter}}, {{person.bio_part}}',
+  '{{word.noun}} {{person.bio_supporter}}, {{person.bio_part}} {{internet.emoji}}',
```
Comment (matthewmayer) [inline src/locales/zh_CN/person/bio_pattern.ts]: Probably remove the space between the noun and the bio_supporter?

File: src/locales/zh_CN/animal/type.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export default ['bear', 'cat', 'dog', 'fish', 'horse'];
```
Comment (matthewmayer) [inline src/locales/zh_CN/animal/type.ts]: Should be translated?

**Review by matthewmayer — CHANGES_REQUESTED** [review]

**Comment by matthewmayer** [comment]
You can mark the pull request as Draft if it's still work in progress.

**Comment by ST-DDT** [comment]
Please stay below 20 files.

**Comment by CoolPlayLin** [comment]
@ST-DDT Done

**Review by matthewmayer — APPROVED** [review]

**Comment by CoolPlayLin** [comment]
@ST-DDT @import-brain

**Comment by ST-DDT** [comment]
> @ST-DDT import-brain

Please be patient. We are currently busy and dont have time to review this PR. Don't worry we haven't forgotten about it and will get to it soon.

File: src/locales/zh_CN/word/adjective.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  '细',
+  '红',
+  '绿',
+  '平坦',
+  '整齐',
+  '雪白',
+  '笔直',
+  '绿油油',
+  '血淋淋',
+  '骨碌碌',
```
Comment (ST-DDT) [inline src/locales/zh_CN/word/adjective.ts:16]: I couldn't find a translation for this. "Gululu"/"Bone Lulu"?

File: src/locales/zh_CN/word/verb.ts [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  '切',
+  '拆',
+  '咬',
+  '吞',
+  '吐',
+  '吮',
+  '吸',
+  '啃',
+  '喝',
+  '咀',
```
Comment (ST-DDT) [inline src/locales/zh_CN/word/verb.ts:23]: Same here. "Tsui"?

File: src/locales/zh_CN/word/verb.ts [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  '撒',
+  '按',
+  '弹',
+  '撞',
+  '提',
+  '扭',
+  '捶',
+  '持',
+  '揍',
+  '披',
```
Comment (ST-DDT) [inline src/locales/zh_CN/word/verb.ts:51]: Same here. "Phi"? Here there are a few alternatives, but they confuse me even more what it actually means.

**Review by ST-DDT — COMMENTED** [review]
I found a few words that I'm unable to check.
Maybe someone with better search or translate skills can enlighten me here.
Or provide a whole sentence/phrase containing them so that the translator can make sense from them?!

**Comment by CoolPlayLin** [comment]
@ST-DDT 
1. 骨碌碌：This term is often used to describe the rapid movement of the spherical object. It also could be a onomatopoeia word
2. 披：
>Wear/Wearing: "披" can mean to wear or wearing, such as wearing a robe or wearing a blanket. For example, "He is wearing a red coat."
>
>Cover/Covering: "披" can also mean to cover or covering, like wearing a cape or letting hair down. For example, "She used a cape to cover her body."
>
>Hang On: "披" can be used to indicate hanging something on the body, such as hanging a sword sheath or hanging a signboard. For example, "The soldier hung his sword sheath in preparation for departure."
>
>Take On: In some idiomatic expressions, "披" can signify taking on a role or responsibility, such as "披荆斩棘" (facing difficulties bravely) or "披星戴月" (enduring hardships). For example, "He bravely faced difficulties and completed the challenging task."
>
>Reveal/Revealing: Sometimes, "披" can also mean revealing or spreading, such as revealing information or spreading a message. For example, "He revealed an important discovery."
>
>Spread/Spreading: In some contexts, "披" can convey the idea of spreading or disseminating, such as spreading news with a cloak or spreading a message. For example, "He spread news far and wide using a cloak."
3. 咀
>Chew: "咀" can mean the action or process of chewing food. For example, "他正在咀东西" can be translated as "He is chewing something," indicating the action of chewing food.
>
>Savor/Contemplate: "咀" can also represent savoring or contemplating a problem, often used in conjunction with "味" (taste). For instance, "他在咀味这个难题" can be translated as "He is contemplating this difficult problem."
>
>Argue/Debate: In certain idiomatic expressions, "咀" can indicate arguing or debating, such as in "咀咒," which means to argue or quarrel. For example, "他们咀咒不休" can be translated as "They are constantly arguing."
>
>Oral Communication: Sometimes, "咀" can also represent oral communication, as seen in "咀咒," where "咀" signifies oral. For instance, "他们之间的咀咒是激烈的" can be translated as "The oral exchange between them is intense."

File: src/locales/zh_CN/word/adjective.ts (hunk 3 above)
Comment (CoolPlayLin) [reply src/locales/zh_CN/word/adjective.ts:16]: This is a Chinese slang, You couldn't translation directly

**Comment by CoolPlayLin** [comment]
I think google it directly is a good way to find meaning
image

**Review by ST-DDT — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**